director/projects/migrations/0003_auto_20180820_2314.py

```diff
@@ -42,7 +42,6 @@ class Migration(migrations.Migration):
     dependencies = [
         migrations.swappable_dependency(settings.AUTH_USER_MODEL),
         ('contenttypes', '0002_remove_content_type_name'),
-        ('checkouts', '0006_auto_20180723_1614'),
```

Comment:
This is fine I suppose, there's not really a good way to delete all the existing tables for an app with a migration, since the migration lives in the app (I'm sure you've figure this out). We'll end up with the checkouts tables hanging around in prod but that's not really a big deal.

Reply:
Thanks for looking at that Ben :pray: